src/types.ts

```diff
@@ -497,21 +497,27 @@ export interface DbPayment {
   totalFees: number;
 }
 
+export interface Payment extends Omit<DbPayment, "id"> {
+  id: number;
+}
+
+export enum PermissionMethod {
+  NOSTR_SIGNMESSAGE = "nostr/signMessage",
+  WEBLN_LISTCHANNELS = "webln/listchannels",
```

Comment:
where is this from? 
we don't have listchannels so far. 

those methods will also vary from connector to connector. LND will have different ones than CLN. 

when a a new method is added to a connector then this needs to be listed in the connector and then also here? And in the future we might remove this whitelist.
the request methods are kinda generic, so we do not necessarily know what will be in there. What do we then list here?